debezium-core/src/main/java/io/debezium/connector/base/ChangeEventQueueContext.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.base;
+
+import java.time.Duration;
+import java.util.function.Supplier;
+
+import io.debezium.annotation.SingleThreadAccess;
+import io.debezium.util.LoggingContext;
+
+public class ChangeEventQueueContext {
```

Comment:
I think `ChangeEventQueueConfig` would be better name

Reply:
Not sure if logging context falls under the config category, but I'm open to renaming it if needed.

Reply:
No, `LoggingContext` is really context, so the name should be kept.

Reply:
Yes please, because in this case this class really configures the the queue, so it should be `ChangeEventQueueConfig`

Reply:
@jpechane Could you please review https://github.com/debezium/debezium-design-documents/pull/19 and share your thoughts so that i can update this PR accordingly.